src/tools/jsondoclint/src/item_kind.rs

```diff
@@ -186,7 +186,7 @@ impl Kind {
             ItemKind::StructField => StructField,
             ItemKind::Trait => Trait,
             ItemKind::TraitAlias => TraitAlias,
-            ItemKind::Typedef => Typedef,
+            ItemKind::TypeAlias => Typedef,
```

Comment:
NIT: Would you mind renaming `Typedef` here for consistency.

Reply:
Sure, not sure why I didn't the first time around 😅